libraries/filter.rb

```diff
@@ -2,11 +2,13 @@
 module Line
   class Filter
     def before(current, args)
-      # Insert a set of lines before each match of the pattern
+      # Insert a set of lines immediately before each match of the pattern - seems like this has limited usefulness.  Comments and blank like would screw it up.
```

Comment:
there's a wee typo here

Reply:
The before and after filters as they stand are unlikely to make the final cut.  I'm trying to figure out something more useful.